src/screens/KeyManagement/NonEVMKeys.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+
+import Grid from '@material-ui/core/Grid'
+import CircularProgress from '@material-ui/core/CircularProgress'
+
+import { NonEVMKeysCard } from './NonEVMKeysCard'
+import { useNonEvmAccountsQuery } from 'src/hooks/queries/useNonEvmAccountsQuery'
+
+const SCHEMAS = {
```

Comment:
Hmm this seems weird, does graphql not provide the typings?

I guess you have to do the backend change first.

Reply:
Can fields also have doc metadata in graphql? If I can get the key name and field names through metadata then that would be great since new tables could be added with core only changes

Reply:
Oh cool, looks like I can use annotations: https://stackoverflow.com/questions/39962867/how-do-i-add-a-description-to-a-field-in-graphql-schema-language

Reply:
ah i see what you are trying to do, trying to build a dynamic table, okay this make sense!

Reply:
> Oh cool, looks like I can use annotations: https://stackoverflow.com/questions/39962867/how-do-i-add-a-description-to-a-field-in-graphql-schema-language

hmm never done it before, let me know what you find haha